src/stars.css

```diff
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 24px;
+
+}
+
+.stars__star4 {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n + 4) {
```

Comment:
What will be if I'll use `stars--3` in HTML?

Reply:
not shure what you do you meen

Reply:
![image](https://user-images.githubusercontent.com/63306833/173509254-5a4cf6af-ce98-4e4a-bee1-f8365b228423.png)

Reply:
Fix this one, and deploy changes